### PR TITLE
Anime Following

### PR DESCRIPTION
**Pull request by du-cki**

### CODE REVIEW

File: cogs/information/anime/anime.py [hunk 1]
```diff
@@ -492,45 +495,39 @@ def list_embed(self) -> discord.Embed | None:
...
-        embed = discord.Embed(colour=self.colour, description="\n".join(progress))
+            information.append(desc)
 
-        if self.list_entry and self.list_entry["updatedAt"]:
-            embed.set_footer(text="Last Updated").timestamp = datetime.datetime.fromtimestamp(
-                self.list_entry["updatedAt"],
-            )
-
-        return embed
+        return discord.Embed(title="People you are following", colour=self.colour, description="\n".join(information))
```
Comment (du-cki) [inline cogs/information/anime/anime.py]: The `title` can yet again be shortened here to something like, "Following", "Followed Users", "Your Connections" w/e. The title seems a bit too unnecessarily long and descriptive.

File: cogs/information/anime/__init__.py [hunk 2]
```diff
@@ -313,15 +320,29 @@ async def search(
...
+        if em := media.list_embed:
+            em.set_image(url=PIXEL_LINE_URL)
+            em.set_thumbnail(url=ctx.author.display_avatar.url)
             embeds.append(em)
 
-        if em := item.following_status:
-            embeds.append(item.following_status)
+        if em := media.following_status_embed(user):
+            em.set_image(url=PIXEL_LINE_URL)
+            embeds.append(em)
```
Comment (du-cki) [inline cogs/information/anime/__init__.py]: These can be left within `Media.from_json` amongst other stuff tbh. Cleans up the command quite a bit.

File: cogs/information/anime/oauth.py [hunk 3]
```diff
@@ -326,12 +330,12 @@ async def get_current_user(self, token: str) -> User:
...
+    async def get_user(self, username: str) -> User | None:
         """Gets a user by their username."""
 
         async with self.session.post(self.URL, json={"query": USER_QUERY, "variables": {"name": username}}) as resp:
             json = await resp.json()
             try:
                 return User.from_json(json["data"]["User"])
             except Exception:
-                raise GenericError("Couldn't find a user with that name.")
+                return None
```
Comment (du-cki) [inline cogs/information/anime/oauth.py:341]: Was raising the error here so the error gets propagated to discord.py and let it go through the bot's error handler, hence simplifying the command code just a bit. But makes sense to revert t his change for consistencies sake.

**Review by itswilliboy — APPROVED** [review]

**Comment by Hadock-is-ok** [comment]
lgtm

**Pull request merged**